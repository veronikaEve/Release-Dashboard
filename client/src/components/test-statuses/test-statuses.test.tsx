import { render } from "@testing-library/react";
import TestStatuses from "./test-statuses";

describe("TestStatuses", () => {
  const props = {
    badge: "img.url",
  };

  test("should match snapshot", () => {
    const { asFragment } = render(<TestStatuses {...props} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
