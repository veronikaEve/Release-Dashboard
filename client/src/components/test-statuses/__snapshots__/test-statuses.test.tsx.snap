// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestStatuses should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="test-status-card text-center card"
  >
    <div
      class="card-header"
    >
      Test Status
    </div>
    <div
      class="card-body"
    >
      <div
        class="info-group row"
      >
        <div
          class="info-group__label col"
        >
          Unit test
        </div>
        <div
          class="info-group__info col"
        >
          <img
            alt="status badge"
            src="img.url"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
